python/ray/serve/scripts.py

```diff
@@ -729,6 +729,22 @@ def status(address: str, name: Optional[str]):
 def shutdown(address: str, yes: bool):
     warn_if_agent_address_set()
 
+    # check if the address is a valid Ray address
+    try:
+        # see what applications are deployed on the cluster
+        details = ServeSubmissionClient(address).get_serve_details()
+        apps = details.get("applications", {})
+        if len(apps) == 0:
+            cli_logger.warning(
+                f"Cannot shutdown Serve applications on {address} because there are no applications deployed."
+            )
+            return
+    except Exception:
+        cli_logger.warning(
+            f"Cannot shutdown Serve applications on {address} because it is not a valid Ray address."
```

Comment:
this error message might be misleading -- the address can be valid but the cluster has already been shut down or the client failed to connect for any other reason

we also shouldn't swallow the exception in those cases, as it might hide the fact that something is misbehaving